website/docs/r/appautoscaling_target.html.markdown

```diff
@@ -18,7 +18,7 @@ Provides an Application AutoScaling ScalableTarget resource.
 resource "aws_appautoscaling_target" "dynamodb_table_read_target" {
   max_capacity       = 100
   min_capacity       = 5
-  resource_id        = "table/tableName"
+  resource_id        = "table/${aws_dynamodb_table.table_name.name}"
```

Comment:
Can you change this and the below reference to `${aws_dynamodb_table.example.name}` please?